src/lib.rs

```diff
@@ -97,6 +97,16 @@
 //!
 //! Check out the [custom_key](https://github.com/AaronErhardt/actix-governor/blob/main/examples/custom_key.rs) example to see how a custom key extractor can be implemented.
 //!
+//! # Customize response error content
+//!
+//! By default, when the rate limit is exceeded the error will show up is `Too many requests, retry in {}s`<br>
+//! and content type it's plaintext, if you want to customize it you need to override the [`response_error_content`] function<br>
```

Comment:
My suggestion to improve readability:

 //! and the content type is plaintext. If you want to customize the message and content type you can override the [`response_error_content`] function.